2020-01-25 02:43:33,589 INFO: Microblog startup [in C:\Users\Raynolds\PycharmProjects\mBlog\app\__init__.py:77]
2020-01-25 02:56:13,325 INFO: Microblog startup [in C:\Users\Raynolds\PycharmProjects\mBlog\app\__init__.py:77]
2020-01-25 03:18:39,662 INFO: Microblog startup [in C:\Users\Raynolds\PycharmProjects\mBlog\app\__init__.py:77]
2020-01-25 03:18:50,805 INFO: Microblog startup [in C:\Users\Raynolds\PycharmProjects\mBlog\app\__init__.py:77]
2020-01-25 03:18:58,006 INFO: Microblog startup [in C:\Users\Raynolds\PycharmProjects\mBlog\app\__init__.py:77]
2020-01-25 03:23:04,689 INFO: Microblog startup [in C:\Users\Raynolds\PycharmProjects\mBlog\app\__init__.py:77]
2020-01-25 03:23:14,687 ERROR: Exception on /index [GET] [in C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\Raynolds\PycharmProjects\mBlog\app\main\routes.py", line 56, in index
    prev_url=prev_url)
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\flask\templating.py", line 140, in render_template
    ctx.app,
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\jinja2\asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\jinja2\environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\Raynolds\PycharmProjects\mBlog\app\templates\base.html", line 155, in template
    {% endif %}
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\jinja2\environment.py", line 1005, in render
    return concat(self.root_render_func(self.new_context(vars)))
  File "C:\Users\Raynolds\PycharmProjects\mBlog\app\templates\index.html", line 14, in root
    <nav aria-label="...">
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\jinja2\environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\jinja2\environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\jinja2\loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\jinja2\environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\Raynolds\PycharmProjects\mBlog\app\templates\base.html", line 155, in template
    {% endif %}
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\jinja2\environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\jinja2\parser.py", line 901, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\jinja2\parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\jinja2\parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\jinja2\parser.py", line 268, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\jinja2\parser.py", line 170, in parse_statements
    self.fail_eof(end_tokens)
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\jinja2\parser.py", line 104, in fail_eof
    return self._fail_ut_eof(None, stack, lineno)
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\jinja2\parser.py", line 90, in _fail_ut_eof
    self.fail(' '.join(message), lineno)
  File "C:\Users\Raynolds\PycharmProjects\mBlog\venv\lib\site-packages\jinja2\parser.py", line 59, in fail
    raise exc(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: Unexpected end of template. Jinja was looking for the following tags: 'endblock'. The innermost block that needs to be closed is 'block'.
